pkgs/tools/misc/riemann-c-client/default.nix

```
@@ -1,27 +1,39 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, file , protobufc }:
+{ lib, stdenv, fetchgit, autoreconfHook, pkg-config, file, protobufc
+,withWolfSSL ? false, wolfssl
+,withGnuTLS ? false, gnutls
+,withJSON ? false, json_c
+}:
 
 stdenv.mkDerivation rec {
   pname = "riemann-c-client";
-  version = "1.10.5";
+  version = "2.1.0";
 
-  src = fetchFromGitHub {
-    owner = "algernon";
-    repo = "riemann-c-client";
+  src = fetchgit {
```

Please consider using this one:

```suggestion
  src = fetchFromGitea {
```